packages/opentelemetry-tracing/test/export/BatchSpanProcessor.test.ts

```diff
@@ -174,4 +174,48 @@ describe('BatchSpanProcessor', () => {
       clock.restore();
     });
   });
+
+  describe('force flush', () => {
+    describe('no waiting spans', () => {
+      it('should call an async callback when flushing is complete', done => {
```

Comment:
Would it be worth adding some assertions that ensure that the spans have been processed by the time the callback is invoked?

Reply:
Sure, I can add those later today

Reply:
I think you meant to put this in the 'spans waiting to flush' section. In any case, that's where I'm putting the test.

Reply:
8aa19eb5bfc8de5e7e0c5565558d4a87abf31bba